src/canal_utils.erl

```diff
@@ -15,6 +18,13 @@ error_msg(Msg) ->
     error_logger:error_report(Msg).
 
 
+-spec error_msg(list(), list()) -> ok.
```

Comment:
`io_lib:format`'s first argument is of type `io:format()` (an alias for `atom() | string() | binary()`).  Does it make sense to change your spec to be in line with the function you call?

Reply:
Fixed. Thank you